app/js/playlist/controllers/PlaylistCtrl.js

```diff
@@ -0,0 +1,113 @@
+"use strict";
+/**
+ * @module   FM.playlist.PlaylistCtrl
+ * @author   SOON_
+ * @requires sn.fm.api
+ * @requires ngRoute
+ */
+angular.module("FM.playlist.PlaylistCtrl", [
+    "FM.api.TracksResource",
+    "FM.api.UsersResource",
+    "FM.api.PlayerQueueResource",
+    "ngRoute"
+])
+/**
+ * @method config
+ * @param  {Provider} $routeProvider
+ */
+.config([
+    "$routeProvider",
+    function ($routeProvider) {
+
+        $routeProvider
+            .when("/", {
+                templateUrl: "partials/playlist.html",
+                controller: "PlaylistCtrl",
+                resolve: {
+                    playlistData: ["PlayerQueueResource", "$route", function (PlayerQueueResource, $route){
+                        return PlayerQueueResource.query($route.current.params).$promise;
+                    }]
+                }
+            });
+
+    }
+])
+/**
+ * @constructor
+ * @class PlaylistCtrl
+ * @param {Object} $scope
+ * @param {Array}  playlistData
+ */
+.controller("PlaylistCtrl", [
+    "$scope",
+    "$q",
+    "TracksResource",
+    "UsersResource",
+    "PlayerQueueResource",
+    "playlistData",
+    function ($scope, $q, TracksResource, UsersResource, PlayerQueueResource, playlistData) {
+
+        /**
+         * @property playlist
+         * @type {Array}
+         */
+        $scope.playlist = playlistData;
+
+        /**
+         * Update `playlist` with queue data from the API
+         * @method refreshPlaylist
+         */
+        $scope.refreshPlaylist = function refreshPlaylistQueue(){
+            PlayerQueueResource.query().$promise
+                .then(function (response){
+                    $scope.playlist = response;
+                });
+        };
+
+        /**
+         * On play event, set playback state variables
+         * refresh playlist if song URI doesn't match the playlist
+         * @method onPlay
+         */
+        $scope.onPlay = function onPlay() {
+            $scope.paused = false;
```

Comment:
This paused property probably isn't needed anymore with the Player/Playlist split